WordPress/src/androidTest/java/org/wordpress/android/mocks/RestClientCustomizableMock.java

```diff
@@ -119,16 +119,20 @@ public RestRequest makeRequest(int method, String url, java.util.Map<String, Str
             return dummyReturnValue;
         }
 
-        try {
-            String data = fileToString(filename);
-            if (data == null) {
-                AppLog.e(T.TESTS, "can't read file: " + filename);
-                return dummyReturnValue;
+        String data = fileToString(filename);
+        if (data == null) {
+            AppLog.e(T.TESTS, "Can't read file: " + filename);
+            if (errorListener != null) {
+                errorListener.onErrorResponse(new VolleyError("Can't read file: " + filename));
```

Comment:
`onErrorResponse` can be an expected call during a test. We should simply raise a RuntimeException here.